test/functional/rpc_misc.py

```diff
@@ -56,9 +56,6 @@ def run_test(self):
 
         self.log.info("test logging rpc and help")
 
-        # Test logging RPC returns the expected number of logging categories.
```

Comment:
ACK on removing this test, it seems kind of unnecessary (just checks that the code is the code, not any kind of correctness).

Reply:
Agree